spec/models/region_spec.rb

```diff
@@ -70,6 +72,14 @@
       facility_region = facility.region
       expect(district.reportable_children).to contain_exactly(facility_region)
     end
+
+    it "works for Bangladesh organizations" do
+      allow(CountryConfig).to receive(:current).and_return(CountryConfig.for(:BD)).at_least(:once)
```

Comment:
Does the `at_least(:once)` do anything here? Seems like it's more intended for use with `expect` than `allow`.

Reply:
Yup, it ensures that the code path we are calling is really getting hit.

If we don't have that, and someone changes the country config check, we could end up with false positives here.